.github/workflows/main.yml

```
@@ -19,16 +19,29 @@ jobs:
     # conrainer with pre-installed mender-artifact and mender-cli tools
     container:
       image: mendersoftware/mender-ci-tools:mender-master
-    name: Upload a dummy Mender artifact
+    name: Deploy a dummy Mender Artifact
     steps:
       # Checkout git repository
       - uses: actions/checkout@v3
       # Download Mender bash client
+      - name: Serup environment
+        run: |
+          apk add openssl jq
+          # secrets are not passed to the runner when a workflow is triggered from a forked repository
+          # https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow
+          if [[ -z "${{ secrets.MENDER_TENANT_TOKEN }}" || "${{ secrets.MENDER_TENANT_TOKEN }}" == "" ]]; then
```

This is the same, `-z` will correctly evaluate to `true` if MENDER_TENANT_TOKEN is `""`
```suggestion
          if [[ -z "${{ secrets.MENDER_TENANT_TOKEN }}" ]]; then
```

---

(same below)